fdbclient/Tracing.actor.cpp

```diff
@@ -376,6 +376,15 @@ Span& Span::operator=(Span&& o) {
 Span::~Span() {
 	if (begin > 0.0 && context.isSampled()) {
 		end = g_network->now();
+		if (!links.empty()) {
```

Comment:
I'm also thinking that this should be implemented in the otel collector. Let me try that approach.

Reply:
If not implemented in the collector, it could be guarded by a knob so we could disable it if/when the visualization platform supports more advanced link aggregation.

Reply:
I managed to implement it in the collector so I removed it from here.